lms/templates/learner_dashboard/course_entitlement.underscore

```diff
@@ -0,0 +1,26 @@
+<div id="change-session-<%-entitlementUUID%>" class="message is-shown">
+    <div class="action-header">
+        <% if (currentSessionId) { %>
+            <%- gettext('To change your session or leave your current session, please select from the following:')%>
+        <% } else { %>
+            <%- gettext('In order to view the course you must select a session:')%>
+        <% } %>
+    </div>
+    <div class="action-controls">
+        <select class="session-select">
+            <% _.each(availableSessions, function(session) { %>
+                <option data-session_id="<%-session.session_id%>">
+                    <%-session.session_dates%>
+                    <% if (session.session_id == currentSessionId) { %>
+                        <%- gettext('  - Currently Enrolled')%>
```

Comment:
I think the original design from @dabdul-hathi specified not to include the current session. Is this possible? If not, we still need to fix the terminology:
"Currently Enrolled" -> "Currently Selected".

Reply:
Agreed there, changed it.